### DISCUSSION TITLE
Does Azure Storage Account settings affect Teams Toolkit Chat App behavior?

### DISCUSSION THREAD

**Discussion by AlbertoPerez8** [OfficeDev/microsoft-365-agents-toolkit#10083] [Q&A]
Currently I have a chat application created with Teams Toolkit & Azure Communications services that's deployed to an Azure storage account and is accessed through Microsoft Teams as a company app. The app and the storage account had been looked over by a team focused on security and some topics had come regarding how some measures could be implemented in order to tighten further security. 

My team and I were told that we should:

- Disable blob anonymous access
- Enable blob soft delete
- Enable public virtual access for selected virtual networks and IP addresses.

![image](https://github.com/OfficeDev/TeamsFx/assets/36648131/b2e4061a-5bfa-496f-8c49-9b148e1237a5)

![image](https://github.com/OfficeDev/TeamsFx/assets/36648131/07e744a8-b1ea-4470-9841-e0a97e738997)

We wanted to know if disabling anonymous access and enabling soft delete would have any effect on how the conversations in the app are being stored, would we see any missing messages or conversations from users? And if we enable virtual access would we have to enable virtual networks / IP addresses for each person using Teams that connects to our app, or does Teams have a IP address range / Virtual Network for this kind of use case?

This application is the first one made by my team using Azure, Teams Toolkit, and the Azure Communications Services SDK, as we work mostly with .NET projects, and wanted to know if going along with said recommendations would affect the behavior of the application or if we can proceed without any issues.

**Comment by MuyangAmigo** [comment]
Hey @AlbertoPerez8 Sorry for the late reply, I'm transferring this discussion to an issue on the Teams platform side so we can track here: https://github.com/MicrosoftDocs/msteams-docs/issues/9775
